doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -279,37 +279,39 @@ configuration option or via the global configuration option. For example:
 
 .. confval:: bluestore_csum_type
 
-inline compression
+Inline Compression
 ==================
 
-bluestore supports inline compression using `snappy`, `zlib`, or
-`lz4`. please note that the `lz4` compression plugin is not
-distributed in the official release.
+BlueStore supports inline compression that uses `snappy`, `zlib`, `lz4`, or
+`zstd`. 
 
-whether data in bluestore is compressed is determined by a combination
-of the *compression mode* and any hints associated with a write
-operation.  the modes are:
+Whether data in BlueStore is compressed is determined by two factors: (1) the
+*compression mode* and (2) any hints associated with a write operation. The
+compression modes are as follows:
 
 * **none**: never compress data.
 * **passive**: do not compress data unless the write operation has a
   *compressible* hint set.
-* **aggressive**: compress data unless the write operation has an
+* **aggressive**: Do compress data unless the write operation has an
   *incompressible* hint set.
 * **force**: try to compress data no matter what.
```

Comment:
Suggest consistent Capitalization or lowercasing of the descriptions above

Reply:
Agreed. So altered.